sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/PortSpec.java

```diff
@@ -68,6 +68,20 @@ public static Builder newBuilder() {
     return new Builder();
   }
 
+  public static Builder newBuilder(PortSpec copy) {
```

Comment:
I think we can get rid of this if this is not used anywhere.

Reply:
Same as above, wanted to leave them in for compleness sake.